packages/client/src/app/views/InfoView.vue

```diff
@@ -0,0 +1,231 @@
+<template>
+  <main>
+    <div class="intro">
+      <h1 class="mb-4-title">Plastic Problemen!</h1>
+      <h3 class="mb-4-subtitle">Leer over plastic met onze Slime!</h3>
+
+      <div class="intro-text">
+        <p class="mb-4 text-markup">
+          Welkom op de website! <br />
```

Comment:
Please check the responsiveness, some texts overlap. You may need to rearrange flex boxes and/or use flex-wrap